MetaMask/metamask-mobile
# refactor: [Part 3] - Ongoing Work for legacy <Text> comp replacement.

## **Description**
This PR replaces old usage of legacy <Text> component with the text component that standardizes on typography provided through theme via [@metamask/design-tokens](https://www.npmjs.com/package/@metamask/design-tokens) library. This will bring uniformity to all text elements on our app.

## **Manual testing steps**
- From the Login screen, enter a wrong password to see the error message.
- Chose Reset Wallet to see the Warning Modal

## **Screenshots/Recordings**

### **Before**
<img width="564" alt="Screenshot 2023-10-23 at 16 33 49" src="https://github.com/MetaMask/metamask-mobile/assets/44679989/65ac96cc-3be4-493d-b726-90de176ed5e5">
<img width="564" alt="Screenshot 2023-10-23 at 16 35 04" src="https://github.com/MetaMask/metamask-mobile/assets/44679989/fba46e4c-8a16-41ed-90fa-153ff0cd2f1b">
<img width="564" alt="Screenshot 2023-10-25 at 10 29 47" src="https://github.com/MetaMask/metamask-mobile/assets/44679989/6da1db3e-cd5b-467b-a02b-54ebe1c3e2ed">



### **After**
<img width="564" alt="Screenshot 2023-10-23 at 16 34 01" src="https://github.com/MetaMask/metamask-mobile/assets/44679989/c3495b78-deb9-404a-a69c-30d7c5a41fb3">
<img width="564" alt="Screenshot 2023-10-23 at 16 49 07" src="https://github.com/MetaMask/metamask-mobile/assets/44679989/3323da11-dcfb-4b1d-ba9c-b408f79fe39e">
<img width="564" alt="Screenshot 2023-10-25 at 10 40 03" src="https://github.com/MetaMask/metamask-mobile/assets/44679989/fc3a4f4c-755b-479a-bf37-43906b97a9df">


## **Related issues**

This is **part** of ongoing tasks related to issue #6887

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
